index.html

```diff
@@ -45,22 +45,316 @@
-                    <div class=""><a href="#"><i class="fa fa-angellist"></i></a></div>
-                </div>
+                <ul class="media">
+                    <li><a href="#"><i class="fa fa-twitter"></i></a></li>
+                    <li><a href="#"><i class="fa fa-linkedin"></i></a></li>
+                    <li><a href="#"><i class="fa fa-medium"></i></a></li>
+                    <li><a href="#"><i class="fa fa-github"></i></a></li>
+                    <li><a href="#"><i class="fa fa-angellist"></i></a></li>
+                </ul>
+                                    
+                <button class="btn btn-one" type="button">Get My Resume</button>
               </div>
+            
+             <div class="col-lg-6">
+                 <ul class="stacks">
+                     <li class="languages">
+                         
+                         <h2> Languages </h2>
+                         <ul class="grid-container">
+                          <li>
+                              
+                              <img src="image/js.png" alt="js_logo">
+                              <p>JavaScript</p>
+                             
+                          </li>
+                          <li>
+                              
+                              <img src="image/html.png" alt="html_logo">
+                              <p>HTML</p>
+                             
+                          </li>
+                             
+                          <li>
+                              
+                              <img src="image/css.png" alt="css_logo">
+                              <p>CSS</p>
+                             
+                          </li>
+                             
+                          
+                        </ul>
+                     </li>
+                     
+                     <li>
+                         <h2> Frameworks </h2>
+                     </li>
+                     
+                      <li>
+                         
+                        <h2> Skills </h2>
+                
+                     </li>
+                 </ul>
+                
+            
+            </div>
          </div>
              
       </div>
   </section>
+      
 
+      
+  <section class="section-contact">
+        <div class="container py-5">
+            
+            <div class="row align-items-center justify-content-center">
+                <div class="col-lg-6">
+                    
+                    <div class="row">
+                         <div class="col-lg-12 col-md-12">
+                             
+                        
+                             <h1>Contact me</h1>
+                    
+                         <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+                         <a class="btn-connect">Let's connect</a>
```

Comment:
- [ ] Just like the previous code review pointed out `Let's connect` should be in a `hx` tag not `a` tag. Consider refactoring this in all lines of code that looks like the example below 👇🏼 

```HTML
    <a class="btn-connect">Let's connect</a>
```